packages/eslint-plugin-template/tests/rules/attributes-order/cases.ts

```diff
@@ -299,4 +303,21 @@ export const invalid = [
            
     `,
   }),
+  convertAnnotatedSourceToFailureCase({
+    messageId,
+    description: 'should work with ng-template',
+    annotatedSource: `
+      <ng-template let-value #Template></ng-template>
+                   ~~~~~~~~~~~~~~~~~~~
+    `,
+    options: [{ alphabetical: true }],
+    data: {
+      expected: '`#Template`, `let-value`',
+      actual: '`let-value`, `#Template`',
+    },
+    annotatedOutput: `
+      <ng-template #Template let-value></ng-template>
+                   
+    `,
+  }),
```

Comment:
👍 I also validated the fixer using this test case with multiple variables and assignment:
```suggestion
  }),
  convertAnnotatedSourceToFailureCase({
    messageId,
    description: 'should work with ng-template with multiple variable assignments',
    annotatedSource: `
      <ng-template let-value="something" let-anotherValue="else" #Template></ng-template>
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    `,
    options: [{ alphabetical: true }],
    data: {
      expected: '`#Template`, `let-anotherValue`, `let-value`',
      actual: '`let-value`, `let-anotherValue`, `#Template`',
    },
    annotatedOutput: `
      <ng-template #Template let-anotherValue="else" let-value="something"></ng-template>
                   
    `,
  }),
```